pages/common/aircrack-ng.md

```diff
@@ -0,0 +1,17 @@
+# aircrack-ng
+
+> Crack WEP and WPA/WPA2 keys from handshake in captured packets.
+> Part of `aircrack-ng`.
```

Comment:
```suggestion
```
Wouldn't the page title already convey that it is `aircrack-ng`?

Reply:
Yes, and i see where the confusion might appear. aircrack-ng is both the name of the software suit this program belongs to (a collection of WIFI tools), and the name of this program itself. The existing pages for aireplay-ng, airodump-ng and airmon-ng (from this collection) all included this line. So for consistency i thought it made sense also to include it here, but let me know what you think?

Reply:
Yeah, that was to reference the suite, I don't think it is required here as it is the same as the program name.

Reply:
This is better if you really want to mention it.
```suggestion
Part of Aircrack-ng network software suite.
```

Reply:
Oops, you missed the `>`, I will update your suggestion.